hal_st/middlewares/ble_middleware/GapSt.hpp

```diff
@@ -30,6 +30,28 @@ namespace hal
             std::array<uint8_t, 16> encryption;
         };
 
+        enum class RfWakeupClock
+        {
+            highSpeedExternal,
+            lowSpeedExternal,
+        };
```

Comment:
```suggestion
        enum class RfWakeupClock : uint8_t
        {
            highSpeedExternal = SHCI_C2_BLE_INIT_CFG_BLE_LS_CLK_HSE_1024,
            lowSpeedExternal = SHCI_C2_BLE_INIT_CFG_BLE_LS_CLK_LSE,
        };
```

Reply:
Above should work as well. Then they are mapped to the definitions in the struct.